api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -61,6 +67,28 @@
           .add("https://www.googleapis.com/auth/cloud-billing")
           .build();
 
+  /**
+   * @return a request scoped cache of firecloud groups, by group name. Groups are lazily populated
+   *     as requested in the cache, and will expire after a duration
+   */
+  @Bean(name = SERVICE_ACCOUNT_GROUP_CACHE)
+  @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
+  public LoadingCache<String, FirecloudManagedGroupWithMembers> firecloudGroupCache(
+      final @Qualifier(SERVICE_ACCOUNT_GROUPS_API) GroupsApi groupsApi) {
+    return CacheBuilder.newBuilder()
+        .expireAfterWrite(30, TimeUnit.SECONDS)
+        // Set a small limit, as groups are large, and our system doesn't use many. If we start
```

Comment:
super nit: it's just RT and CT, right?  so we could go lower

Reply:
agreed, in practice I expect 2 elements - I just wanted to make sure this doesn't accidentally cause problems if the usage of this significantly changes; but setting a higher capacity I believe is free. 10 felt high enough that we wouldn't be close to hitting it if we added more tiers or a few more groups per environment for any reason.